tgui/packages/tgui/components/Image.js

```diff
@@ -4,9 +4,11 @@ import { Box } from "./Box";
 export const Image = (props) => {
   const { pixelated, className, ...rest } = props;
 
-  return (<Box as="img" {...rest} className={classes(
-    "Image",
-    pixelated && "Image--pixelated",
```

Comment:
why not just fix the css?

Reply:
since the talk with style, since apparently fixing the css doesn't work - just remove the css portion